cpp/search/search.cpp

```diff
@@ -2049,6 +2053,11 @@ bool Search::isAllowedRootMove(Loc moveLoc) const {
        (rootSafeArea[moveLoc] == opp || rootSafeArea[moveLoc] == rootPla))
       return false;
   }
+
+  if(searchParams.rootSearchSymmetriesReduce && isSymDupLoc[moveLoc]){
```

Comment:
I think the code is correct already, but just to be safe and explicit, let's make sure it is not Board::PASS_LOC before checking the isSymDupLoc array.

Reply:
Done